File: HaberUygulamaKotlin/app/src/main/java/com/example/haberuygulama/model/YorumModel.kt
package com.example.haberuygulama.model

data class YorumModel(
    val id: Int,
    val haber_id: Int,
    val takma_ad: String,
    val yorum_metni: String,
    val onayli: Int,
    val yorum_tarihi: String
)

File: haber_uygulama_react_native/src/model/yorum_model.ts
export type YorumModel = {
    id: number;
    haber_id: number;
    takma_ad: string;
    yorum_metni: string;
    onayli: number;
    yorum_tarihi: string;
};

export type YorumInsertRequest = {
    haber_id: number;
    takma_ad: string;
    yorum_metni: string;
};
